docs/developers-guide/cadl-docs/README.md

```diff
@@ -0,0 +1,119 @@
+# Morphir Type Mappings To CADL
+## SDK Types to CADL
+####### NB: CADL has a numeric type which consist of two subtypes called integer and float
+
+#### Basic Types
+###### Bool
+    This maps to the `boolean` type in CADL. 
+    Example: 
+        alias InCadl = boolean
+###### Int
+    In CADL, this maps to the subtype `integer`, which further has theses subtypes ::  
+    [-] int8        [-] uint8       [-] safeint
+    [-] int16       [-] uint16
+    [-] int32       [-] unit32
+    [-] int64       [-] unit32
+    
+    NB - To assign an `integer` type, you need to actually assign a subtype instead else the `integer` type defaults to an object. 
+
+    Example: 
+        alias length = int32 
+###### Float
+    In CADL, this maps to the subtype `float`, which further has theses subtypes :  
+    [-] float32   
+    [-] float64
+
+    NB:: To assign an `float` type, you need to actually assign a subtype instead else the `float` type defaults to an object.
+    
+    Example: 
+        alias PI = float32
```

Comment:
Can you check if this is accurate based on https://package.elm-lang.org/packages/elm/core/latest/Basics#Float